access_application.go

```diff
@@ -76,6 +77,92 @@ type AccessApplicationCorsHeaders struct {
 	MaxAge           int      `json:"max_age,omitempty"`
 }
 
+// AccessApplicationScimConfig represents the configuration for provisioning to an Access Application via SCIM.
+type AccessApplicationScimConfig struct {
+	Enabled            bool                                     `json:"enabled"`
+	RemoteURI          string                                   `json:"remote_uri,omitempty"`
+	Authentication     *AccessApplicationScimAuthenticationJson `json:"authentication,omitempty"`
+	IdpUid             string                                   `json:"idp_uid,omitempty"`
+	DeactivateOnDelete bool                                     `json:"deactivate_on_delete"`
+	Mappings           []*AccessApplicationScimMapping          `json:"mappings,omitempty" validate:"unique=Schema,dive"`
```

Comment:
where is the `validate` struct tag used here?

Reply:
missed removing that, thanks